.rubocop_todo.yml

```diff
@@ -1,2 +1,2 @@
-Style/HashSyntax:
-  UseHashRocketsWithSymbolValues: true
+#Style/HashSyntax:
```

Comment:
can we remove this cop from the todo

Reply:
@jstraw can we not remove this completely from the todo now you've corrected it? Correct me if im wrong.